spec/fixtures/opensource_accounts_endpoint.txt

```diff
@@ -1,11 +0,0 @@
-[
```

Comment:
Did you mean to delete all these fixtures?

Reply:
Yeah, we aren't using them anymore. Instead, we're putting the fixtures inline in the tests so it's more clear what the inputs/outputs are.